src/node_modules.cc

```diff
@@ -344,6 +359,28 @@ void BindingData::GetNearestParentPackageJSONType(
   args.GetReturnValue().Set(Array::New(realm->isolate(), values, 3));
 }
 
+void BindingData::GetPackageJSONScripts(
+    const FunctionCallbackInfo<Value>& args) {
+  Realm* realm = Realm::GetCurrent(args);
+  std::string path = "package.json";
+
+  THROW_IF_INSUFFICIENT_PERMISSIONS(
+      realm->env(), permission::PermissionScope::kFileSystemRead, path);
+
+  auto package_json = GetPackageJSON(realm, path);
+  if (package_json == nullptr) {
+    printf("Can't read package.json\n");
+    return;
+  } else if (!package_json->scripts.has_value()) {
+    printf("Package.json scripts doesn't exist or parseable\n");
```

Comment:
```suggestion
    printf("package.json scripts doesn't exist or isn't parseable\n");
```

Reply:
Correct grammar would be `package.json "scripts" object doesn't exist or isn't parseable`